Autopilot/Simulation/Intercepts/CommsBetweenStateMachines_Intercept.cpp

```diff
@@ -0,0 +1,67 @@
+#include "CommWithPathManager.hpp"
+#include "CommWithAttitudeManager.hpp"
+
+#include <fstream>
+
+/***********************************************************************************************************************
+ * Variables
+ **********************************************************************************************************************/
+
+static CommandsForAM commandsForAM = {};
+static AttitudeData attitudeData = {};
+
+static std::fstream PathToAttComLog;
+
+/***********************************************************************************************************************
+ * Prototypes
+ **********************************************************************************************************************/
+
+static void writeAttitudeCommandsToFile(CommandsForAM *commands);
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+
+void CommWithAMInit()
+{
+    PathToAttComLog.open("DebugLogs/PathToAttComLog.txt", std::fstream::out);
+    PathToAttComLog << "";
```

Comment:
Maybe write "Initializing comms"?

Reply:
Do you mean to the terminal ?

Reply:
Yeah, to the same logging file/terminal that the rest of the stuff is being logged to

Reply:
Ah I see. this particular stuff gets logged to the DebugLogs/PathToAttComLog.txt file and that gets cleared on every run, so not too much use in writing the initialize thing.